main/tracking/courseLog.php

```diff
@@ -206,6 +206,32 @@ function(index) {
 Session::write('additional_user_profile_info', $userProfileInfo);
 Session::write('extra_field_info', $extra_info);
 
+$defaultExtraFields = [];
+$defaultExtraFieldsFromSettings = [];
+$defaultExtraFieldsFromSettings = api_get_configuration_value('course_log_default_extra_fields');
+if (isset($defaultExtraFieldsFromSettings)) {
+    $default_extra_fields = $defaultExtraFieldsFromSettings;
```

Comment:
Use camelCase for  $default_extra_fields and $user_array

Reply:
Variables names changed to camelCase